bessctl/conf/testing/run_module_tests.bess

```diff
@@ -156,15 +153,19 @@ def monitor_task(module, wid):
 def run_tests():
     DID_TESTS_FAIL = False
 
-    path = os.path.dirname(os.path.realpath(sys.argv[0]))
+    # Env variable used to run a subset of tests
+    test_name = $TEST_NAME!'*'
+
+    default_path = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])),
+                                'conf/testing/module_tests')
+    test_path = $TEST_PATH!default_path
```

Comment:
At some point we might want to modify this to allow more than one path name component (split at `:` like `$PATH`, perhaps) so that if you have several plugin directories you can point to all of them at once.  But again, this should be fine for now.

Reply:
Unless you want to address this one in this PR @sangjinhan, I'll merge now